.rubocop_todo.yml

```diff
@@ -1,15 +1,11 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 500`
-
-# on 2019-07-20 01:16:04 -0500 using RuboCop version 0.72.0.
+# on 2019-07-23 14:13:22 -0400 using RuboCop version 0.65.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 1
-# Configuration parameters: MinNameLength, AllowNamesEndingInNumbers, AllowedNames, ForbiddenNames.
-# AllowedNames: io, id, to, by, on, in, at, ip, db
-Naming/UncommunicativeMethodParamName:
+Style/FrozenStringLiteralComment:
   Exclude:
-    - 'app/pb_kits/playbook/pb_list/list.rb'
+    - 'app/pb_kits/playbook/pb_bar_graph/bar_graph.rb'
```

Comment:
To prevent rubocop errors for List, this needs to stay as:
```
- 'app/pb_kits/playbook/pb_list/list.rb'
```

If you have an offense in `'app/pb_kits/playbook/pb_bar_graph/bar_graph.rb'`, please fix

Reply:
No idea how that got that way. But i fixed it just now. Good catch!